resources/js/components/password-input.tsx

```
@@ -0,0 +1,33 @@
+  const [visible, setVisible] = useState(false)
+
+  return (
+    <div className="relative">
+      <Input
+        type={visible ? "text" : "password"}
+        className={`pe-9 sm:pe-8 ${className ?? ""}`.trim()}
+        {...props}
+      />
+      <button
+        type="button"
+        className="absolute end-0 top-0 flex h-full items-center px-2.5 text-muted-fg hover:text-fg transition-colors"
```

Biome (2.4.12)

[error] These CSS classes should be sorted.

(lint/nursery/useSortedClasses)